vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/LinkManager.java

```diff
@@ -216,6 +217,11 @@ public void deleteExternalLink(String linkId) {
         return new ArrayList<>(spaceLinkDisplayRepo.findSpaceLinkDisplaysForSpace(spaceId));
     }
 
+    @Override
+    public List<ISpaceLinkDisplay> getSpaceLinkDisplaysToPublishedSpace(String spaceId, SpaceStatus spaceStatus){
```

Comment:
same as before (I believe), does it get displays for links to published spaces or for the given space status? method name and parameters don't match up.